pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+```
+
+```bash
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386601388,"method":"GET","url":"/batch/transaction/index/139","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386601861,"method":"GET","url":"/batch/transaction/index/140","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386602362,"method":"GET","url":"/batch/transaction/index/141","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386602915,"method":"GET","url":"/batch/transaction/latest","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386602923,"method":"GET","url":"/batch/transaction/latest","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386602930,"method":"GET","url":"/batch/transaction/index/142","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386603499,"method":"GET","url":"/batch/transaction/index/143","elapsed":15,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386604104,"method":"GET","url":"/batch/transaction/index/144","elapsed":3,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386604635,"method":"GET","url":"/batch/transaction/index/145","elapsed":4,"msg":"Served HTTP Request"}
+simple-optimism-node-dtl-1  | {"level":30,"time":1674386605243,"method":"GET","url":"/batch/transaction/index/146","elapsed":3,"msg":"Served HTTP Request"}
+```
+
+#### Verify fault-detector logs 
+```bash
+docker compose logs fault-detector -f
+```
+```bash
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386539264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.639 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386554265,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.836 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386569266,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.624 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386584265,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.928 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386599264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.452 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386614265,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.352 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386629264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.516 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386644264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.639 ms\n","msg":"server log"}
+```
+
+## Verify
+
+#### Method 1:
```

Comment:
What's the difference between the methods? When should I choose one?

Reply:
Any method will work, depends upon the user which way they prefer it, one will get the latest block height through a RPC call and the other one through the geth ipc console.